### PR TITLE
Corrected Chassis Over Temp oid for state indexes

### PR DESCRIPTION
**Pull request by craig-nokia**
Updated the discovery timos.yaml file to correct the state_translations table for the over temperature alarm.

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/definitions/discovery/timos.yaml [hunk 1]
```diff
@@ -82,7 +82,7 @@ modules:
                         - { value: 8, generic: 0, graph: 0, descr: GreenBlink }
                         - { value: 9, generic: 2, graph: 0, descr: RedBlink }
                 -
-                    oid: tmnxChassisTable
+                    oid: tmnxChassisOverTempState
```
Comment (PipoCanaja) [inline includes/definitions/discovery/timos.yaml:85]: Hi
What was wrong with the initial oid ? Looks good according to the MIB file at least.

Comment (craig-nokia) [reply includes/definitions/discovery/timos.yaml:85]: See if I can explain it...

The tmnxChassisTable has 10 different states for alarms and in the same table, the tmnxChassisOverTempState has only 2 states.  When discovery happens, it overwrites the values 1 and 2 in the state translation table.

Before:  The Critical LED would say "Overtemp"
image
With the patch, the correct alarm text is displayed "Red :
image

Comment (PipoCanaja) [reply includes/definitions/discovery/timos.yaml:85]: Got it.
The issue should be solved in a different way. 
Just keep current OID and VALUE, and add this statement to ensure the state is correct for this entry. : 
` state_name: tmnxChassisOverTempState `
Basically, any state from this table without a `state_name` defined should be corrected.

**Review by PipoCanaja — CHANGES_REQUESTED** [review]

**Comment by PipoCanaja** [comment]
Could you check the current YAML file ?

**Comment by craig-nokia** [comment]
Thanks PipoCanaja!   Adding the state_names to the oid: tmnxChassisTable is good to go for the timos.yaml file.

**Review by PipoCanaja — APPROVED** [review]
LGTM

**Pull request merged**